extensions/spring-xd-extension-gpload/src/main/java/org/springframework/xd/gpload/GploadEnvironmentProvider.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.xd.gpload;
+
+import org.springframework.batch.step.tasklet.x.EnvironmentProvider;
+
+import java.util.Map;
+
+/**
+ * Class that sets environment variables for gpload
+ */
+public class GploadEnvironmentProvider implements EnvironmentProvider {
+
+	String gploadHome;
+
+	boolean usePasswordFile;
+
+	String passwordInfo;
+
+	public GploadEnvironmentProvider(String gploadHome, boolean usePasswordFile, String passwordInfo) {
+		this.gploadHome = gploadHome;
+		this.usePasswordFile = usePasswordFile;
+		this.passwordInfo = passwordInfo;
+	}
+
+	@Override
+	public void setEnvironment(Map<String, String> env) {
+		env.put("GPHOME_LOADERS", gploadHome);
```

Comment:
Don't do this because it will result errors in system which don't have this. I have a GPDB demo vm's and its `gpload` is:
```
$ cat /usr/local/greenplum-db/bin/gpload
#!/bin/sh
if [ ! -z "$GPHOME" ]; then
    . $GPHOME/greenplum_path.sh
fi
if [ ! -z "$GPHOME_LOADERS" ]; then
    . $GPHOME_LOADERS/greenplum_loaders_path.sh
fi
if [ `uname -s` = "AIX" -a ! -z "$GP_LIBPATH_FOR_PYTHON" ]; then
    LIBPATH=$GP_LIBPATH_FOR_PYTHON:$LIBPATH
    export LIBPATH
fi
```

Don't have `greenplum_loaders_path.sh` and job then fails with:
```
2015-04-08 03:29:14,432 1.2.0.SNAP ERROR task-scheduler-3 gpload.GploadTasklet - gpload job finished with exit code: 1
2015-04-08 03:29:14,434 1.2.0.SNAP ERROR task-scheduler-3 gpload.GploadTasklet - gpload err: /usr/local/greenplum-db/bin/gpload: line 6: /usr/local/greenplum-db/greenplum_loaders_path.sh: No such file or directory
```